tests/playlists.py

```diff
@@ -776,6 +776,18 @@
 2.aac
 '''
 
+CUE_OUT_WITH_DURATION_KEY_PLAYLIST = '''#EXTM3U
```

Comment:
This doesn't appear to get used in a test anywhere ...

Reply:
Updated parser.py "_parse_cueout"

Reply:
I didn't include a test as it didn't seem needed, similar to _cueout_simple.  I included it for reference on the notation.